fe/src/main/java/org/apache/doris/load/loadv2/SparkEtlJobHandler.java

```diff
@@ -0,0 +1,156 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.load.loadv2;
+
+import org.apache.doris.PaloFe;
+import org.apache.doris.common.LoadException;
+import org.apache.doris.common.util.Util;
+import org.apache.doris.load.EtlStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.apache.spark.launcher.SparkAppHandle;
+import org.apache.spark.launcher.SparkAppHandle.Listener;
+import org.apache.spark.launcher.SparkLauncher;
+
+import com.google.common.collect.Maps;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Map;
+
+public class SparkEtlJobHandler {
+    private static final Logger LOG = LogManager.getLogger(SparkEtlJobHandler.class);
+
+    private static final String JOB_CONFIG_DIR = PaloFe.DORIS_HOME_DIR + "/temp/job_conf";
+    private static final String JOB_CONFIG_FILE = "jobconfig.json";
+    private static final String APP_RESOURCE = PaloFe.DORIS_HOME_DIR + "/lib/spark-etl-1.0.0.jar";
+    private static final String MAIN_CLASS = "org.apache.doris.dpp.Dpp";
+    private static final String ETL_JOB_NAME = "doris__%s";
+    // hdfs://host:port/outputPath/dbId/loadLabel/PendingTaskSignature
+    private static final String ETL_OUTPUT_PATH = "%s/%s/%d/%s/%d";
+
+    class SparkAppListener implements Listener {
+        @Override
+        public void stateChanged(SparkAppHandle sparkAppHandle) {
+            sparkAppHandle.getAppId();
+            sparkAppHandle.getState();
+        }
+
+        @Override
+        public void infoChanged(SparkAppHandle sparkAppHandle) {}
+    }
+
+    public SparkAppHandle submitEtlJob(long loadJobId, String loadLabel, String sparkMaster,
+                                       Map<String, String> sparkConfigs, String jobJsonConfig) throws LoadException {
+        // create job config file
+        String configDirPath = JOB_CONFIG_DIR + "/" + loadJobId;
+        String configFilePath = configDirPath + "/" + JOB_CONFIG_FILE;
+        try {
+            createJobConfigFile(configDirPath, configFilePath, loadJobId, jobJsonConfig);
```

Comment:
Here we need add a ```todo```, for clearing temporary file produced by SparkLoadJob. Such as config, or hive table